DataFormats/L1TParticleFlow/interface/PFCandidate.h

```diff
@@ -0,0 +1,61 @@
+#ifndef DataFormats_L1TParticleFlow_PFCandidate_h
+#define DataFormats_L1TParticleFlow_PFCandidate_h
+
+#include <vector>
+#include "DataFormats/L1Trigger/interface/L1Candidate.h"
+#include "DataFormats/L1Trigger/interface/Muon.h"
+#include "DataFormats/L1TParticleFlow/interface/PFCluster.h"
+#include "DataFormats/L1TParticleFlow/interface/PFTrack.h"
+
+namespace l1t {
```

Comment:
don't you want to use the same namespace as in https://github.com/cms-sw/cmssw/pull/29348/files#diff-4ad75913d7e9668f34e61ae8aa206c80R12 (ie. `l1tp2`)?

Reply:
no.  l1tp2 used in case of CalorimeterPhase2 is to distinguish those Calo objects from the Phase1 Calo objects. 
In this case, there are no PF objects in Phase1, so "l1t" namespace suffices.